### PR TITLE
Makes acid melt flares faster

### PR DESCRIPTION
**Pull request by Zar000**
## About The Pull Request
Exactly what the title says

## Why It's Good For The Game

Marines have a lot of tools to light up areas, some of them, such as howie flares & CAS flares have almost no counter-play other than waiting them out.

The thrown flares however, we can melt but it takes forever unless queen / boiler does it.

This still makes it take a while with drone level acid, but 2x faster.
## Changelog
:cl:
balance: All acids melt flares 2x faster
/:cl:

### CODE REVIEW

**Comment by docRoboRobert** [comment]
after the flare boxes massive buff i'd say this is needed.

File: code/game/objects/effects/aliens.dm [hunk 1]
```diff
@@ -168,7 +170,10 @@
 		return
 	if(loc != acid_t.loc && !isturf(acid_t))
 		loc = acid_t.loc
-	ticks += delta_time * acid_strength
+	if(istype(acid_t, /obj/item/flashlight/flare))
+		ticks += delta_time * min((acid_strength * flare_strength_multiplier), flare_strength_max_melt_rate) // we dont wanna make it possible to melt flares too fast
```
Comment (Lumipharon) [inline code/game/objects/effects/aliens.dm:174]: Having the typecheck here is snowflake. There should be a proc on the target side to return an acid time or whatever.

**Review by Lumipharon — CHANGES_REQUESTED** [review]

**Comment by Tailiik** [comment]
It would be very funne if a primo prae could trigger flare melt with ranged spit

**Review by Lumipharon — CHANGES_REQUESTED** [review]
Make this a proc instead of an object var as discussed.

**Comment by Mannybrado** [comment]
maintainers help

**Pull request closed without merging**